metrics/test_driver/cli/selenium/driver/chrome_driver.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:io';
+
+import 'package:archive/archive.dart';
+
+import '../../../common/config/driver_tests_config.dart';
+import '../../../util/file_utils.dart';
+
+/// Represents the chrome driver.
+class ChromeDriver {
+  /// Check if the chrome driver file exists in [workingDir].
+  /// If not - downloads the chrome driver and makes it executable.
+  static Future<void> prepare(String workingDir) async {
```

Comment:
This does nothing if chromeDriverFile exists, am I right? Is it correct that the process is started only if the file does not exist?